framework/CodeInterfaces/CobraTF/CTFinterface.py

```diff
@@ -0,0 +1,98 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default',DeprecationWarning)
+
+import os
+from ctfdata import ctfdata
+from CodeInterfaceBaseClass import CodeInterfaceBase
+
+class CobraTF(CodeInterfaceBase):
+    def createNewInput(self, currentInputFiles, oriInputFiles, samplerType ,**Kwargs):
+        # @ In, Kwargs
+        # @ Out, current input file names & modified variables list
+	from CTFparser import CTFparser
+
+        # currently, CTF parsers only work with point sampler type
+        if 'dynamicevent' in samplerType.lower():
+        	raise IOError("Dynamic Even Tree-based sampling not implemented yet!")
```

Comment:
indentation in our code is 2 spaces. Here there are 6 and are not consistent in the all code. 
Please fix